ionos_sbx/grafana/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  artifacthub.io/license: Apache-2.0
+  artifacthub.io/links: |
```

Comment:
1. I think we already have Grafana on SBX.

Reply:
2. Commits are not prefixed with env name

Reply:
You're right, I noticed that Grafana is deployed as part of the monitoring tools, however, my account does not have the permissions to create dashboards. I'll try to contact "groundnuty" to ask if they can grant me the necessary privileges or at least point me to the right person. Alternatively, we might need to set up a separate Grafana instance.

Reply:
Please contact Michal in order to get the permissions. In the meantime I have asked hin for comment on this MR.